.idea/google-java-format.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="GoogleJavaFormatSettings">
+    <option name="enabled" value="false" />
```

Comment:
@goelsonali what do you think of us using google format, we can also adjust the code format if we want to.

Reply:
i think ideas are good enough to format the code, I think we don't need these format settings.
But would like to explore SONAR option how to configure SONAR and run default checksum rules.

Reply:
yep, we can create issues for that and take care soon